include/nuttx/tls.h

```diff
@@ -304,7 +304,7 @@ uintptr_t task_tls_get_value(int tlsindex);
 
 #if defined(up_tls_info)
 #  define tls_get_info() up_tls_info()
-#elif defined(CONFIG_TLS_ALIGNED)
+if defined(CONFIG_TLS_ALIGNED) && !defined(__KERNEL__)
```

Comment:
sorry, I made a typo error.:(, it's strange that the compiler can pass without error.